platform/broadcom/platform-modules-accton.mk

```diff
@@ -8,6 +8,7 @@ ACCTON_AS7312_54X_PLATFORM_MODULE_VERSION = 1.1
 ACCTON_AS7326_56X_PLATFORM_MODULE_VERSION = 1.1
 ACCTON_AS7716_32XB_PLATFORM_MODULE_VERSION = 1.1
 ACCTON_AS6712_32X_PLATFORM_MODULE_VERSION = 1.1
+ACCTON_AS7726_32X_PLATFORM_MODULE_VERSION = 1.1
```

Comment:
644

Reply:
OK. I send "abdb2ff" commit to file mode to 644